extension/mergeSort.js

```diff
@@ -0,0 +1,38 @@
+function mergeSort( initialSet = [] ){
+  if( initialSet == null ){
+    return 'Cannot sort null'
+  }
+  if( initialSet.constructor !== Array ){
+    return 'Please use Arrays'
+  }
```

Comment:
We can condense the number of check we do at the beginning of this function:

``` if( !Array.isArray(initialSet) ){  /* This will let us know if the set is an array or not.*/ }```